src/backend/storeManagers/legendary/library.ts

```diff
@@ -634,3 +637,42 @@ export async function runRunnerCommand(
     }
   )
 }
+
+export async function getLegendaryApi(): Promise<ResponseDataLegendaryAPI> {
+  const response = await axios.get<ResponseDataLegendaryAPI>(
```

Comment:
maybe we should wrap this in a try/catch? if for some reason the request fails this will throw an exception